lib/archethic/bootstrap/network_init.ex

```diff
@@ -139,6 +141,29 @@ defmodule Archethic.Bootstrap.NetworkInit do
     |> self_replication()
   end
 
+  @spec init_network_reward_pool() :: :ok
+  def init_network_reward_pool() do
+    Logger.info("Create mining reward pool")
+
+    data = %TransactionData{
+      content: """
+      initial supply: #{@genesis_network_pool_amount}
+      """
+    }
+
+    tx = %Transaction{address: address} = Transaction.new_nft_reward(data)
+
+    tx
+    |> self_validation([
+      %UnspentOutput{
+        from: address,
+        amount: @genesis_network_pool_amount,
+        type: {:NFT, address}
```

Comment:
Are reward tokens are non-fungible NFT's?
If they are fungible , we might need to have nft_id 0. as per aeip2

Reply:
This is an old commit, go on the last to see the new token standard implementation https://github.com/archethic-foundation/archethic-node/pull/393/commits/c9c195a45d0c894b63108318b8bd9764c92388bf